src/components/tag/tag.js

```diff
@@ -3,62 +3,88 @@ import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import { css } from '@emotion/core';
 import vars from '../../../materials/custom-properties';
+import designTokens from '../../../materials/design-tokens';
 import Constraints from '../constraints';
 import AccessibleButton from '../buttons/accessible-button';
 import Text from '../typography/text';
 import { CloseBoldIcon } from '../icons';
 
-const getTextDetailColor = isDisabled => {
-  if (isDisabled) return vars.colorGray60;
-  return vars.colorBlack;
+const getTextDetailColor = (isDisabled, theme) => {
+  const overwrittenVars = {
+    ...vars,
+    ...theme,
+  };
+  if (isDisabled)
+    return overwrittenVars[designTokens.fontColorForTagWhenDisabled];
+  return overwrittenVars[designTokens.fontColorForTag];
 };
 
-const getContentWrapperStyles = props => css`
-  display: flex;
-  box-sizing: border-box;
-  align-items: center;
-  min-height: ${vars.sizeHeightTag};
-  border-radius: ${vars.borderRadiusTag};
-  padding: 5px ${vars.spacing8};
-  cursor: default;
-  font-family: inherit;
-  white-space: normal;
-  text-align: left;
-  min-width: 0;
-  overflow-wrap: break-word;
-  hyphens: auto;
-  border-style: solid;
-  border-width: 1px;
-  border-color: ${props.type === 'warning'
-    ? vars.borderColorTagWarning
-    : vars.borderColorTagPristine};
+const getContentWrapperStyles = (props, theme) => {
+  const overwrittenVars = {
+    ...vars,
+    ...theme,
+  };
```

Comment:
Q: do we allow all custom properties to be overwritten or should we only allow design tokens?

Reply:
That's something I've thought about. The idea is that (for now), users can override `choices`. And since our designTokens use `choices`, that's kind of how it's working together for now.

However, there's some other variables that we could allow users to override as well, but I think it would make sense to rename them first.

#### UI Kit wide variables

example:
```
spacing8
```

These are often named in a way that would make overriding them weird. Spacing8 is set to 16? 😕 . We would probably want to rename these.

#### Component specific variables
These are sort of like "magic" numbers. If we allow users to override them, it should be documented in the individual components.

``
sizeHeightTag
standardInputHeight
```

Reply:
#644 to track this

Reply:
Right, the spacings tokens should probably be something like `spacingsXs`, `spacingsS`, etc